packages/firebase_auth/firebase_auth_platform_interface/lib/src/method_channel/method_channel_user.dart

```diff
@@ -30,29 +40,21 @@ class MethodChannelUser extends UserPlatform {
   @override
   Future<void> delete() async {
     try {
-      await MethodChannelFirebaseAuth.channel.invokeMethod<void>(
-        'User#delete',
-        _withChannelDefaults({}),
-      );
+      await _api.delete(pigeonDefault);
     } catch (e, stack) {
       convertPlatformException(e, stack);
     }
   }
 
   @override
-  Future<String> getIdToken(bool forceRefresh) async {
+  Future<String?> getIdToken(bool forceRefresh) async {
```

Comment:
Breaking: Is now nullable because the underlying token can be null (was forced unwrapped before)